app/controllers/categories_controller.rb

```diff
@@ -9,4 +9,19 @@ def index
     categories = categories.where(top_level: top_level) unless top_level.nil?
     render json: CategoryPresenter.present(categories)
   end
+
+  # rubocop:disable Metrics/AbcSize
+  def counts
+    if !admin_signed_in?
+      render status: :unauthorized
```

Comment:
We should consider refactoring this into a [`before_action`](http://guides.rubyonrails.org/action_controller_overview.html#filters), but probably in a later PR.

Reply:
Yea, good point.